frontend/app/src/components/StreamlitDialog/StreamlitDialog.test.tsx

```diff
@@ -102,7 +115,10 @@ describe("aboutDialog", () => {
         })}
       </Fragment>
     )
-    expect(wrapper.find(Modal).exists()).toBe(true)
-    expect(wrapper.find(Modal).text()).not.toContain("Streamlit v")
+
```

Comment:
Think we should still check that a modal renders

Reply:
done